neurons/validators/validator.py

```diff
@@ -81,6 +88,14 @@ def __init__(self, config=None):
         
         super(Validator, self).__init__(config)
 
+        # Init API and start
+        if config.enable_api:
+            # external requests
+            self.api_server = APIServer(
```

Comment:
maybe pass here metagraph object instance? it should be synced by default validator logic that we have allready